client/src/redux/reducers/workbench.reducers.js

```diff
@@ -1,13 +1,21 @@
 export const initialState = {
-  debug: false,
+  loading: false,
```

Comment:
WORKBENCH_INIT isn't being used currently.  what about dropping it and just having the `loading` set to true at the start.  
```suggestion
  loading: true,
```

Reply:
🤦 clearly not at my best at 2am. 

hmm I think I prefer invoking the spinner since we don't set `loading` to `true` as a default anywhere else. or is there precedent?

Reply:
it is kind of a weird thing compared to the other `loading`'s. but either way is fine (starting at state indicating spinner or getting to that state using WORKBENCH_INIT).  

should `loading` be renamed to something `hasWorkbenchConfiguration`?

Reply:
lets keep it as is